rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/aws/config/RoscoAWSConfiguration.groovy

```diff
@@ -63,6 +65,7 @@ class RoscoAWSConfiguration {
     List<AWSVirtualizationSettings> virtualizationSettings = []
   }
 
+  @AutoClone(style = AutoCloneStyle.SIMPLE)
```

Comment:
Tried (and failed) to use `@Immutable`. Wasn't able to figure out how to convince spring to initialize the beans from the properties without the fields of `GCEVirtualizationSettings` and `AWSVirtualizationSettings` ending up `null`. Guessing it relates to the way I'm initializing `GCEBakeryDefaults` and `AWSBakeryDefaults`.